### ISSUE TITLE
Presentation keyboard navigation doesn't work in KDE

### ISSUE THREAD

**Issue by fwsmit** [mozilla/pdf.js#15487]
Moving to the next page with teh keyboard doesn't work. 

Attach (recommended) or Link to PDF file here: any pdf

Configuration:
- Web browser and its version: firefox-104.0.2-1.fc36.x86_64
- Operating system and its version: fedora 36 (KDE)
- PDF.js version: Don't know
- Is a browser extension: No

Steps to reproduce the problem:
1. Open a pdf
2. Open presentation mode
3. Try to go to the next page with the right arrow

What is the expected behavior?

It goes to the next page

What went wrong?

It stays on the same page

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):
In the online viewer this works. It's only broken in the firefox built-in viewer.

**Comment by Snuffleupagus** [comment]
Duplicate of https://bugzilla.mozilla.org/show_bug.cgi?id=1787456, which was uplifted to Firefox 105.

**Comment by fwsmit** [comment]
Thanks, can confirm it's fixed for me in Firefox 105
